changelog.yaml

```diff
@@ -51,6 +51,13 @@
 - version: NEXT
   date: TBD
   changes:
+     - type: bug
+       impact: patch
+       title: Fix deletion bug #241
+       description: |-
+         When a pipeline run was deleted the state and the result has not been updated. This is fixed now.
+       upgradeNotes: `ResultDeleted` was only used in the metrics before. Now it can also occur as a pipeline run result.
```

Comment:
```suggestion
       upgradeNotes: |-
         `ResultDeleted` was only used in the metrics before. Now it can also occur as a pipeline run result.
```